files/usr/share/jeos-firstboot/modules/raspberrywifi

```diff
@@ -68,20 +76,27 @@ do
     if [ "${#list[@]}" -eq "2" ]; then
         wlan_device="${list[0]}"
     else
-        d --menu  $"Select wireless card to configure" 0 0 $dh_menu "${list[@]}"
+        d --menu  $"Select wireless card to configure" 0 0 "$(menuheight ${#list[@]})" "${list[@]}"
         wlan_device="${result}"
     fi
 
-    if raspberrywifi_get_wlan_networks && d --menu  $"Select wireless network to connect" 0 0 $dh_menu "${list[@]}"; then
+    if raspberrywifi_get_wlan_networks && d --no-tags --menu  $"Select wireless network to connect" 0 0 "$(menuheight ${#list[@]})" "${list[@]}"; then
         wlan_network="$result"
     else
         if raspberrywifi_wlan_error $"Error listing wireless networks"; then
             continue
         fi
         break
     fi
-
-    d --menu $"Select authentication mode" 0 0 $dh_menu $"WPA-PSK" '' $"WPA-EAP" '' $"Open" ''
+	if [ "$wlan_network" == "manual" ]; then
```

Comment:
Broken indentation

Reply:
tabs vs spaces again, I will fix this all at once, changing everything to tabs in the new code.